rllib/agents/sac/sac_policy.py

```diff
@@ -165,9 +172,8 @@ def actor_critic_loss(policy, model, _, train_batch):
     q_t_selected = tf.squeeze(q_t, axis=len(q_t.shape) - 1)
     if policy.config["twin_q"]:
         twin_q_t_selected = tf.squeeze(twin_q_t, axis=len(q_t.shape) - 1)
-        q_tp1 = tf.minimum(q_tp1, twin_q_tp1)
-
-    q_tp1 -= tf.expand_dims(alpha * log_pis_t, 1)
+        q_tp1 = tf.reduce_min((q_tp1, twin_q_tp1), axis=0)
```

Comment:
Just to make sure: both of these result in the same `q_tp1`? I think this part of code will fail silently if one of the dimensions end up being `(N, )` instead of `(N, 1)`.

Reply:
Just checked on pdb and both q_tp1 and twin_q_tp1 are (N,1). The policy graphs for softlearning and Rllib SAC are pretty much identical. Debugged SAC by setting random seeds to constant values and made everything deterministic. Softlearning and Rllib SAC output the same loss function values even after multiple gradient steps...so I believe any other deltas could be from code outside of SAC.

Reply:
Nice, sounds good!